app/src/main/java/com/duckduckgo/app/survey/ui/SurveyActivity.kt

```diff
@@ -146,13 +149,21 @@ class SurveyActivity : DuckDuckGoActivity() {
         fun intent(
             context: Context,
             survey: Survey,
+            source: SurveySource,
         ): Intent {
             val intent = Intent(context, SurveyActivity::class.java)
             intent.putExtra(SURVEY_EXTRA, survey)
+            intent.putExtra(SURVEY_SOURCE_EXTRA, source)
             return intent
         }
 
         const val SURVEY_EXTRA = "SURVEY_EXTRA"
+        const val SURVEY_SOURCE_EXTRA = "SURVEY_SOURCE_EXTRA"
+
+        enum class SurveySource {
+            PUSH,
+            IN_APP,
+        }
```

Comment:
Now that we were touching this, it'd have been great to move it to use the `GlobalActivityStarter`

Reply:
Follow-up task -> https://app.asana.com/0/0/1204888830157412/f